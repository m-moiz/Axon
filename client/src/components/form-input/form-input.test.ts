import React from 'react';
import { shallow } from 'enzyme';
import FormInput from './form-input.component';

it('renders FormInput Component', () => {
	expect(shallow(<FormInput />)).toMatchSnapshot();
});
