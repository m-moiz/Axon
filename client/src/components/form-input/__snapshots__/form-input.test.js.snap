// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders FormInput Component 1`] = `ShallowWrapper {}`;
